tox.ini

```diff
@@ -310,7 +310,6 @@ passenv =
     TWINE_PASSWORD
 commands =
     {[testenv:build]commands}
-    twine upload --skip-existing {toxinidir}/dist/*
```

Comment:
@farleyb-amazon Does there need to be a command here? Sorry, my Tox is weak, but in other spots there are commands after the {[*]} nonesense.